pkg/gen/input/workflows/internal/file/create_release_pr.go

```diff
@@ -39,26 +39,72 @@ jobs:
           cat <<EOF
           ${{ toJson(github) }}
           EOF
+  gather_facts:
+    name: Gather facts
+    runs-on: ubuntu-18.04
+    outputs:
+      unchanged: ${{ steps.get_unchanged.outputs.unchanged }}
+      base: ${{ steps.get_base.outputs.base }}
+      version: ${{ steps.get_version.outputs.version }}
+    steps:
+      - name: Get base
+        id: get_base
+        run: |
+          base="$(echo ${{ github.event.ref }} | cut -d '#' -f 1)"
+          echo "base=\"$base\""
+          echo "::set-output name=base::${base}"
+      - name: Get version
+        id: get_version
+        run: |
+          version="$(echo ${{ github.event.ref }} | cut -d '#' -f 3)"
+          version="${version#v}" # Strip "v" prefix.
+          echo "version=\"$version\""
+          echo "::set-output name=version::${version}"
+      - name: Checkout base code
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ steps.get_base.outputs.base }}
+      - name: Get unchanged
+        id: get_unchanged
+        env:
+          sha: ${{ github.sha }}
+        run: |
+          base_head=$(git rev-parse HEAD)
+          echo "base_head=\"$base_head\""
+          unchanged="false"
+          if [[ "$base_head" == "${{ env.sha }}" ]] ; then
+            unchanged="true"
+          fi
+          echo "unchanged=\"$unchanged\""
+          echo "::set-output name=unchanged::${unchanged}"
```

Comment:
> but then we need to figure a way to not create a PR when it's already created.

This is done here. Base ref if checked out and we compare base head with current ref. If they are equal we set `unchanged` to true.